backend/src/main/java/vn/edu/hust/nmcnpm_20242_n3/entity/BookRequest.java

```diff
@@ -24,6 +24,12 @@ public class BookRequest {
     @ManyToOne(cascade = CascadeType.ALL)
     BookLoan bookLoan;
 
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
This relationship should be `ManyToOne`, as canceled requests or requests that has been fulfilled can also be linked to a `BookCopy`.